src/components/PageHeadingFlowRuns.vue

```diff
@@ -1,19 +1,34 @@
 <template>
   <page-heading class="page-heading-flow-runs" :crumbs="crumbs">
     <template v-if="!hideActions" #actions>
-      <SavedFilters />
+      <SavedFilters v-model:filter="filter" />
     </template>
   </page-heading>
 </template>
 
 <script lang="ts" setup>
+  import { computed } from 'vue'
   import PageHeading from '@/components/PageHeading.vue'
   import SavedFilters from '@/components/SavedFilters.vue'
+  import { SavedSearchFilter } from '@/models/SavedSearch'
 
-
-  defineProps<{
+  const props = defineProps<{
+    filter: SavedSearchFilter,
```

Comment:
I'm surprised to see this as a prop here - feels like a not-ideal drill/emit situation. If the heading doesn't do anything but re-emit events from the `SavedFilters` component it might save some headache to expose the `#actions` slot for consumers to handle themselves.

Reply:
Generally speaking I agree prop drilling and event bubbling isn't a great pattern. But this felt better to me so that the heading component could remain a single component for implementation. Otherwise each project would need to add the heading and add the correct actions. This way its all included and there's no chance for oss and cloud to be different on accident.